app/package.json

```diff
@@ -35,7 +35,7 @@
     "testcafe": "^1.18.4"
   },
   "scripts": {
-    "start": "meteor --no-release-check --exclude-archs web.browser.legacy,web.cordova --settings ../config/settings.development.json",
+    "start": "meteor --port 3004 --no-release-check --exclude-archs web.browser.legacy,web.cordova --settings ../config/settings.development.json",
```

Comment:
@devineng @htobin @YangQiantwx @balemayehu1 if you don't mind, is it ok to develop on port 3004 instead of 3000 ?